Ports/SuperTuxKart/package.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=SuperTuxKart
+useconfigure='true'
+version='1.4'
+archive_hash='9890392419baf4715313f14d5ad60746f276eed36eb580636caf44e2532c0f03'
+files="https://github.com/supertuxkart/stk-code/releases/download/${version}/supertuxkart-${version}-src.tar.xz ${port}-${version}-src.tar.xz $archive_hash"
+auth_type='sha256'
+workdir="${port}-${version}-src"
+launcher_name='SuperTuxKart'
+launcher_category='Games'
+launcher_command='/usr/local/bin/supertuxkart'
+depends=(
+    'curl'
+    'freetype'
+    'glu'
+    'harfbuzz'
+    'libjpeg'
+    'libopenal'
+    'libogg'
+    'libpng'
+    'libvorbis'
+    'openssl'
+    'SDL2'
+    'sqlite'
+    'zlib'
+)
+
+configopts=(
+    '-DCMAKE_BUILD_TYPE=STKRelease'
+    "-DCMAKE_INSTALL_PREFIX=${SERENITY_INSTALL_ROOT}/opt/SuperTuxKart"
+    "-DSTK_INSTALL_DATA_DIR=${SERENITY_INSTALL_ROOT}/opt/SuperTuxKart"
+    '-DSUPERTUXKART_DATADIR=/opt/SuperTuxKart'
+    "-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
+    '-DBUILD_RECORDER=OFF'
+    '-DBUILD_SHARED_LIBS=OFF'
+    '-DUSE_WIIUSE=OFF'
+    '-DUSE_DNS_C=ON'
+    '-Bbuild'
+    '-GNinja'
+    "-DCMAKE_C_FLAGS=-I${SERENITY_INSTALL_ROOT}/usr/include/LibGL -I${SERENITY_INSTALL_ROOT}/usr/local/include/SDL2 -I${SERENITY_INSTALL_ROOT}/usr/local/include -lcurl -lharfbuzz -lfreetype"
+    "-DCMAKE_CXX_FLAGS=-I${SERENITY_INSTALL_ROOT}/usr/include/LibGL -I${SERENITY_INSTALL_ROOT}/usr/local/include/SDL2 -I${SERENITY_INSTALL_ROOT}/usr/local/include -lcurl -lharfbuzz -lfreetype"
+    '-DUSE_IPV6=OFF'
+    '-DUSE_SYSTEM_ENET=OFF'
+)
+
+configure() {
+    run cmake "${configopts[@]}" .
+}
+
+build() {
+	cd SuperTuxKart-1.4-src/build
+	cp "${SERENITY_INSTALL_ROOT}/usr/local/lib/libcurl.so" "${SERENITY_INSTALL_ROOT}/usr/lib/libcurl.so"
+	cp "${SERENITY_INSTALL_ROOT}/usr/local/lib/libharfbuzz*" "${SERENITY_INSTALL_ROOT}/usr/lib/"
+	ninja
+}
+
+install() {
+	cd SuperTuxKart-1.4-src/build
```

Comment:
There's a couple of tabs here while most of the file (and ports in general) uses space, let's fix those up :)